v23.1/connect-to-a-changefeed-kafka-sink-with-oauth-using-okta.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Connect to a Changefeed Kafka sink with OAuth Using Okta
+summary: Learn how to set up OAuth through Okta for a changefeed streaming to Kafka.
+toc: true
+docs_area: stream_data
+---
+
+{% include_cached new-in.html version="v23.1" %} CockroachDB {{ site.data.products.enterprise }} changefeeds can stream change data out to [Apache Kafka](https://kafka.apache.org/) using OAuth authentication.
```

Comment:
Can we link this back to more basic changefeed related detail?

Maybe just link the initial word changefeeds to 
 change-data-capture-overview.html , since that page, in turn, links to the Enterprise details?

Reply:
Yes!